### PR TITLE
Minor cleanups prior release 3.0

### PR DESCRIPTION
**Pull request by bsipocz**
The numbers for PRs already include the 3 still open ones (including this one).

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -267,10 +267,6 @@ astropy.io.ascii
 astropy.io.fits
 ^^^^^^^^^^^^^^^
 
-- Fix bug of dropping table column meta (format, description, meta) when writing a table
```
Comment (eteq) [inline CHANGES.rst]: why was this deleted?  It's currently in v3.0.x...?

Comment (bsipocz) [reply CHANGES.rst]: as it was fixing a bug that was also introduced in 3.0.0, see slack discussion

Comment (eteq) [reply CHANGES.rst]: nm, I saw a slack message about this (it was changed to afffects-dev when judged to go into 3.0.0 instead of 3.0.1

Comment (eteq) [reply CHANGES.rst]: Hah, came to it at the same time...

**Comment by eteq** [comment]
(I manually killed the non-docs tests because for some reason they were running)

**Comment by eteq** [comment]
Something weird is happening wrt the travis status and github, but https://travis-ci.org/astropy/astropy/builds/339514696 reveals the docs passed here, so I'm merging.
